lib/generator.js

```diff
@@ -207,6 +207,108 @@ class Generator {
       await this.generateDirectoryStructure(this.asyncapi);
       await this.launchHook('generate:after');
     }
+  } */
+
+  // Main method for generating code based on AsyncAPI document
```

Comment:
Please switch to the jsdoc comments (for all functions) and use the same as the old one for the function 🙂